addOns/replacer/src/main/javahelp/org/zaproxy/zap/extension/replacer/resources/help/contents/replacer.html

```diff
@@ -61,6 +61,30 @@ <h3>Replacement String</h3>
 Hexadecimal bytes are represented with "\x00 - \xFF".
 The literal string "\x00" can be kept with "\\x00".
 
```

Comment:
Add a paragraph.

Reply:
This part I hadn't changed from the original code but since I'm at it anyways would you like to see paragraphs around the rest of the documentation paragraphs that are currently bare in this file?

Reply:
I don't think it's necessary to change other parts in this PR, if you want open other PR to tidy up help sounds good to me.